testapi.pm

```diff
@@ -60,7 +60,7 @@ our @EXPORT = qw(
 
   wait_screen_change assert_screen_change wait_still_screen assert_still_screen wait_serial
   record_soft_failure record_info force_soft_failure
-  become_root x11_start_program ensure_installed eject_cd power
+  become_root x11_start_program ensure_installed eject_cd disconnect_usb power
```

Comment:
How about


```suggestion
  become_root x11_start_program ensure_installed eject_cd disconnect_device power
```

which is a bit more generic

Reply:
I was kinda following `eject_cd` which isn't really just for ejecting CDs, but I like how the name makes it clear what it's *usually* for. If we can think of more cases where we'd maybe want to disconnect different devices, maybe the generic name makes more sense?

Reply:
well, thinking about it maybe usb is generic enough as "usb" is already much more generic than "cd" ;) So keep it